pilot/pkg/features/pilot.go

```diff
@@ -321,4 +321,11 @@ var (
 		"If this is set to true, one Istiod will control remote clusters including CA.").Get()
 	EnableCAServer = env.RegisterBoolVar("ENABLE_CA_SERVER", true,
 		"If this is set to false, will not create CA server in istiod.").Get()
+
+	EnableTProxyPreserveSourceIP = env.RegisterBoolVar(
```

Comment:
As I commented before, please remove this option.
Unconditionally use the original_src filter when in the TPROXY mode. Don't make this optional.
I don't see any downside to doing this.

Reply:
Option has been removed, thanks!